website/skaffold.yaml

```diff
@@ -18,6 +18,7 @@ build:
           APP_VERSION: staging-{{.GITHUB_SHA}}
           DEPLOY_ENVIRONMENT: staging
           IMPORT_SERVER_URL: https://staging.snackager.expo.io
+          LEGACY_SNACK_SERVER_URL: https://staging.snack.expo.dev
```

Comment:
In order to test the redirects on staging, I think we should set this to `staging.snack.expo.io`